docs/src/main/sphinx/release/release-352.md

```diff
@@ -0,0 +1,88 @@
+* Allow prepared statement parameters for `SHOW STATS`. ({issue}`6582`)
+* Update tzdata version to 2020d. As a result, queries can no longer reference 
+  `US/Pacific-New` zone, as it has been removed. ({issue}`6660`)
+* Improve performance of queries using {func}`rank()` window function. ({issue}`6333`)
+* Improve performance of {func}`sum` and {func}`avg` for `decimal` types. ({issue}`6951`)
+* Improve join performance. ({issue}`5981`)
+* Improve query planning time. ({issue}`6544`)
+* Fix window and streaming aggregation semantics regarding peer rows. Now peer rows are
+  not distinct from each other, while previously they had to been equal. ({issue}`6472`) 
+* Fix query failure when using element of `array(timestamp(p))` in a complex expression 
+  for `p` greater than 6. ({issue}`6350`)
+
+## Web UI
+
+* Support JWK with certificate chain for OAuth2 authorization. ({issue}`6428`)
+* Optionally verify JWT audience (`aud`) field for OAuth2 Web UI authentication. ({issue}`6501`)
+* Add scopes to `OAuth2` configuration. ({issue}`6580`)
+* Prevent from replay attacks in `OAuth2` by using `nonce` cookie when `openid` scope is requested. ({issue}`6580`)
+
+## JDBC Driver
+
+* Support user impersonation when using password-based authentication. This is configured 
+  using `sessionUser` connection URL parameter. Previously user impersonation was possible
+  only with other authentication mechanisms. ({issue}`6549`)
+* Add OAuth2 authentication support.({issue}`6576`)
+
+## Docker Image
+
+* Remove support for configuration directory `/usr/lib/trino/etc`. The configuration should be provided in `/etc/trino`.
+
+## CLI
+
+* Support user impersonation when using password-based authentication. This is configured
+  using `--session-user` command line parameter. Previously user impersonation was possible 
+  only with other authentication mechanisms. ({issue}`6567`)
+
+## BigQuery Connector
+
+* Fix query failure when executing `count(*)` without `WHERE` clause for views. ({issue}`6706`)
+
+## Hive Connector
+
+* Support reading timestamp with microsecond or nanosecond precision (as configured with 
+  `hive.timestamp-precision` property) nested within `array`, `map` or `struct` data types. ({issue}`5195`)
+* Support reading from table in Sequencefile format that uses LZO compression. ({issue}`6452`)
+* Expose AWS HTTP Client stats via JMX. ({issue}`6503`)
+
+## Iceberg Connector
+
+* Optimize cardinality-insensitive aggregations (max, min, distinct, approx_distinct) over 
+  identity partition columns with `optimizer.optimize-metadata-queries` config property 
+  or `optimize_metadata_queries` session property. ({issue}`5199`)
+* Provide  `use_file_size_from_metadata` catalog session property and `iceberg.use-file-size-from-metadata` 
+  config property to fix query failures on tables with wrong file sizes stored in the metadata. ({issue}`6369`)
+ 
+## Kinesis Connector
+
+* Support GZIP message compression. ({issue}`6442`)
```

Comment:
@guyco33 @kokosing  does https://github.com/trinodb/trino/commit/539e84604a7bb843c9ae0872a35af46cee03c82e / https://github.com/trinodb/trino/pull/6705 add support for "UNCOMPRESSED", or is it a refactor only?

Reply:
It's a refactor only - just to preserve the current behaviour